styles/styles.css

```diff
@@ -166,6 +169,9 @@
   --space-13px: 0.813rem;
   --space-14px: 0.875rem;
   --space-15px: 0.938rem;
+
+  /* negative margin */
+  --space-negative-15px: -0.938rem;
```

Comment:
If usage is rare we can use something like
`margin: calc(-1 * var(--space-15px));`

Reply:
make sense. done